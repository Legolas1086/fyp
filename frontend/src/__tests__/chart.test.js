import { render, screen } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Chat from '../card'

describe('snapshot testing',()=>{
    test("renders approve the request compnenet",()=>{
        const fragment = render(
            <BrowserRouter>
            <Chat />
            </BrowserRouter>
        ).asFragment();
    expect(fragment).toMatchSnapshot()
    })
});