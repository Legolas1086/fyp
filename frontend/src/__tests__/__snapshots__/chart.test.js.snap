// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing renders approve the request compnenet 1`] = `
<DocumentFragment>
  <div
    class="d-flex justify-content-around"
  >
    <div
      class="card"
      style="width: 20rem; height: 16rem;"
    >
      <div
        class="card-body"
      >
        <img
          class="card-img-top"
          src="Ngo.jpg"
        />
        <div
          class="card-title h5"
        >
          NGOs
        </div>
      </div>
      <div
        class="card-footer"
      >
        <a
          class="card-link"
          href="/avlngo"
        >
          Click here
        </a>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
